draft-ietf-httpbis-client-hints.md

```diff
@@ -117,11 +117,11 @@ However, traditional proactive content negotiation techniques often mean that cl
 
 This document defines a new response header, Accept-CH, that allows an origin server to explicitly ask that clients send these headers in requests. It also defines guidelines for content negotiation mechanisms that use it, colloquially referred to as Client Hints.
 
-Client Hints mitigate the performance concerns by assuring that clients will only send the request headers when they're actually going to be used, and the privacy concerns of passive fingerprinting by requiring explicit opt-in and disclosure of required headers by the server through the use of the Accept-CH response header.
+Client Hints mitigate performance concerns by assuring that clients will only send the request headers when they're actually going to be used, and privacy concerns of passive fingerprinting by requiring explicit opt-in and disclosure of required headers by the server through the use of the Accept-CH response header.
 
-This document defines the Client Hints infrastructure, a framework that enables servers to opt-in to specific proactive content negotiation features, which will enable them to adapt their content accordingly. However, it does not define any specific features that will use that infrastructure. Those features will be defined in their respective specifications.
+This document defines Client Hints, a framework that enables servers to opt-in to specific proactive content negotiation features, adapting their content accordingly. However, it does not define any specific features that will use that infrastructure. Those features will be defined in their respective specifications.
 
-One example of such a feature is the User Agent Client Hints feature {{UA-CH}}.
+One example of such a feature is the User Agent Client Hints feature {{!UA-CH}}.
```

Comment:
! makes it a normative ref

Reply:
Oops. So your comment was to remove the reference altogether? Or can I keep it as informative?

Reply:
Informative is probably OK.

Reply:
Done!